plugin/pkg/scheduler/factory/factory.go

```diff
@@ -191,6 +194,35 @@ func NewConfigFactory(client clientset.Interface, schedulerName string, hardPodA
 	return c
 }
 
+// GetNodeStore provides the cache to the nodes.  This won't be used often outside the scheduler, except in mock tests
+// which make fake nodes.  Otherwise, the nodes will be maintained internally by the cache itself.
+// Mostly this is an internal function only used in rare external circumstances (i.e. testing)
```

Comment:
Can you simplify the comment just to "For testing purposes only." ? Or sth like that?